actions/user_actions.jsx

```diff
@@ -40,10 +40,15 @@ export async function switchFromLdapToEmail(email, password, token, ldapPassword
     }
 }
 
-export async function loadProfilesAndTeamMembers(page, perPage, teamId = getCurrentTeamId(getState()), success) {
```

Comment:
Can you remove the success callback and have callers use `.then(success)` instead?

Reply:
Since we're passing it to `loadTeamMembersForProfilesList` and you're not changing that function right now, we can get around that for now by explicitly returning an explicit Promise instead of using an async function for the body of the action like
```javascript
function doSomething() {
    return (dispatch, getState) => {
        return new Promise((resolve) => {
            loadTeamMembersForProfilesList(resolve);
        });
    };
);
```

Reply:
I'm not really surprised. We probably just added success and error for consistency with all the other actions

Reply:
If it is working properly, I'm ok with it, but looks like this is not exactly the same behavior under the hood. Here, the promise is resolved when the `getProfilesInTeam` is success, in the previous implementation it the success call was made after the `loadTeamMembersForProfilesList`.

Reply:
It's working properly after testing several cases, and yes, it's a bit different since it only dispatches  `loadTeamMembersForProfilesList` and `loadStatusesForProfilesList` after success call in `getProfilesInTeam` which I believe makes sense.
The `loadProfilesAndTeamMembers` return `{data: true}` which can be consumed when a callback is necessary.